packages/world/src/systems/libraries/TerrainLib.sol

```diff
@@ -15,7 +15,7 @@ library TerrainLib {
 
   /// @notice Get the terrain block type of a voxel coordinate.
   /// @dev Assumes to be called from a root system.
-  function _getBlockType(VoxelCoord memory coord) internal view returns (uint8) {
+  function _getBlockType(VoxelCoord memory coord) public view returns (uint8) {
```

Comment:
are you hitting contract size limits?

Reply:
yup! when I set it to internal both build system and mine system hit limits